docs/swagger.yaml

```diff
@@ -124,54 +98,86 @@ components:
           example: '   https://api.vipps.no/dwo-api-application/v1/deeplink/vippsgateway?v=2&token=eyJraWQiO....'
       required:
         - url
-    QR_Response:
-      title: QR_Response
+    OneTimePaymentQrResponse:
+      title: OneTimePaymentQrResponse
       type: object
+      x-examples: {}
+      description: ''
       properties:
         url:
           type: string
           description: Link to QR image
+          example: 'https://qr.vipps.no/generate/qr.png?...'
         expiresIn:
           type: integer
           description: How many seconds more this QR will be active
-    Error:
-      title: Error
+          example: 544
+      required:
+        - url
+        - expiresIn
+    QrErrorResponse:
+      title: QrErrorResponse
       type: object
+      x-examples:
+        example-1:
+          title: Order timeout
+          detail: Payment has already timed out
+          instance: 3cff3564-4a14-4481-8dbb-431087753eed
+        example-2:
+          title: Validation error
+          detail: Supplied landing page token is not valid
+          instance: 3cff3564-4a14-4481-8dbb-431087753eed
+        example-3:
+          title: Validation error
+          detail: Merchant supplied id is not valid
+          instance: 3cff3564-4a14-4481-8dbb-431087753eed
       properties:
         title:
           type: string
           description: Title for the error
+          example: Validation error
         detail:
           type: string
           description: Detailed description of the error
+          example: Supplied landing page token is not valid
         instance:
           type: string
           description: UUID for the request
+          example: 3cff3564-4a14-4481-8dbb-431087753eed
     MerchantRedirectQrRequest:
       title: MerchantRedirectQrRequest
       type: object
+      x-examples: {}
       properties:
         id:
           type: string
+          description: Merchant supplied Id for QR
+          example: billboard_1
           nullable: true
-        url:
+        redirectUrl:
```

Comment:
@Sigvedash @khenrichsen Should we discuss this a little bit more? I think we all disagree of what to call this.